tests/test262-esnext-excludelist.xml

```diff
@@ -5295,32 +5295,8 @@
   <test id="built-ins/FinalizationRegistry/target-not-callable-throws.js"><reason></reason></test>
   <test id="built-ins/FinalizationRegistry/undefined-newtarget-throws.js"><reason></reason></test>
   <test id="built-ins/FinalizationRegistry/unnaffected-by-poisoned-cleanupCallback.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/constructor.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/instance-extensible.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/length.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/name.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/newtarget-prototype-is-not-object.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/prop-desc.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/proto-from-ctor-realm.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/proto.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/prototype-from-newtarget-abrupt.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/prototype-from-newtarget-custom.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/prototype-from-newtarget.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/prototype/Symbol.toStringTag.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/prototype/constructor.js"><reason></reason></test>
-  <test id="built-ins/WeakRef/prototype/deref/custom-this.js"><reason></reason></test>
   <test id="built-ins/WeakRef/prototype/deref/gc-cleanup-not-prevented-with-wr-deref.js"><reason></reason></test>
```

Comment:
Do we know why these tests fail?

Reply:
These two tests need other feature to work. For example Finalization Registry